lib/r10k/module_loader/puppetfile.rb

```diff
@@ -113,7 +111,10 @@ def add_module_metadata(name, info)
 
       # @param [String] forge
       def set_forge(forge)
-        @forge = forge
+        if @allow_puppetfile_forge
+          logger.debug _("Using Forge from Puppetfile: %{forge}") % { forge: forge }
```

Comment:
Should we also log that we're ignoring the `forge` setting from the Puppetfile if `@allow_puppetfile_forge` isn't true?

Reply:
I could go either way, it's default behavior and has been that way a really long time. But I can add it.

Reply:
Do we think we'd want to make this the default behavior in the next major release? If so, I think it could be good to log a message about it. ¯\_(ツ)_/¯ 
Otherwise, if we think this will be opt-in forever, I'm good with leaving it out.

Reply:
I seriously doubt we will ever make this the default behavior. We're leaning away from having people put more things in their puppetfiles.

Reply:
In that case, I'm good with leaving out the logging. (sorry for the back and forth 😅 )

Reply:
Having tested this out, I think I'll leave it in, actually. There's isn't any other logging currently at `debug` or `debug1` that tells you which forge you're using.